resin/auth.py

```diff
@@ -119,6 +148,12 @@ def get_token(self):
         Raises:
             InvalidOption: if not logged in and there is no token in Settings.
 
+        Examples:
+            # If you are logged in.
```

Comment:
@dfunckt This doesn't look like other examples because this is a block of code so `>>>` is omitted. I think we don't need to prepend `>>>` to the comment line, a comment line should start with `#`